### ISSUE TITLE
MultipartRelated Start

### ISSUE THREAD

**Issue by fenzad** [jstedfast/MimeKit#124]
The code line:

ContentType.Parameters["start"] = "<" + value.ContentId + ">";

is right?
Thunderbird with this parameter with the characters < and > shows an empty message!

**Comment by jstedfast** [comment]
From rfc2387, we have this:

```
3.4.  Syntax

     related-param   := [ ";" "start" "=" cid ]
                        [ ";" "start-info"  "="
                           ( cid-list / value ) ]
                        [ ";" "type"  "=" type "/" subtype ]
                        ; order independent

     cid-list        := cid cid-list

     cid             := msg-id     ; c.f. [822]
```

Then, in rfc822, it defines `msg-id` as:

```
     msg-id      =  "<" addr-spec ">"            ; Unique message id
```

So it looks like Thunderbird is broken.

**Comment by fenzad** [comment]
In the rfc 822 i can find this:
___
3.2.  The Start Parameter

   The start parameter, if given, is the content-ID of the compound
   object's "root".  If not present the "root" is the first body part in
   the Multipart/Related entity.  The "root" is the element the
   applications processes first.
___

So in 99 % of cases (including mine), the parameter is not necessary. If the parameter is not specified the message is shown correctly in Thunderbird.

**Comment by jstedfast** [comment]
Yep, I know it's not necessary. That doesn't make it not a bug in Thunderbird, though :-)

**Comment by jstedfast** [comment]
Can you file a bug on https://bugzilla.mozilla.org about the Thunderbird bug? Thanks.

**Comment by fenzad** [comment]
The bug already exists https://bugzilla.mozilla.org/show_bug.cgi?id=471402
from 2008...how sad :-(

thanks anyway, but in my condition I guess I'll have to edit and recompile the source code of mimekit.

**Comment by jstedfast** [comment]
I've added a work-around to not use a start parameter if it is the first part in the multipart. But shame on Thunderbird for not adhering to the specifications.

**Comment by fenzad** [comment]
Fantastico!

**Comment by jstedfast** [comment]
Probably not until this weekend when I have time to make a release.

**Comment by fenzad** [comment]
Ok ... the weekend is a reasonable time. 
I'm happy to wait!
Thank you very much!

**Comment by fenzad** [comment]
Hi...when a new release in nuget?

**Comment by jstedfast** [comment]
Sorry about that, I got distracted this past weekend. There will be a release this weekend for sure.

**Comment by fenzad** [comment]
Thank you very much!

**Comment by jstedfast** [comment]
I've released MimeKit 1.0.13 with this fix.
